core/src/main/java/io/seata/core/rpc/grpc/service/TransactionManagerService.java

```diff
@@ -0,0 +1,264 @@
+    }
+
+    @Override
+    public void globalReport(GlobalReportRequestProto request, StreamObserver<GlobalReportResponseProto> responseObserver) {
+        GlobalReportRequest requestModel = (GlobalReportRequest) ProtoTypeConvertHelper.convertToModel(request);
+        RpcMessageHandleContext handleContext = buildHandleContext(responseObserver);
+        handleContext.setMessageReply(response -> {
+            if (!(response instanceof GlobalReportResponse)) {
+                LOGGER.warn("[GRPC]wrong response type, need {} but actually {}", GlobalReportResponse.class, response.getClass());
+                return;
+            }
+            try {
+                responseObserver.onNext((GlobalReportResponseProto) ProtoTypeConvertHelper.convertToProto(response));
+                responseObserver.onCompleted();
+            } catch (Exception e) {
+                LOGGER.warn("[GRPC]fail to send response, req:{}, resp:{}", request, response);
+            }
+        });
+        remotingServer.processMessage(handleContext, requestModel);
+    }
+
+    @Override
+    public void getGlobalStatus(GlobalStatusRequestProto request, StreamObserver<GlobalStatusResponseProto> responseObserver) {
+        GlobalStatusRequest requestModel = (GlobalStatusRequest) ProtoTypeConvertHelper.convertToModel(request);
+        RpcMessageHandleContext handleContext = buildHandleContext(responseObserver);
+        handleContext.setMessageReply(response -> {
+            if (!(response instanceof GlobalStatusResponse)) {
+                LOGGER.warn("[GRPC]wrong response type, need {} but actually {}", GlobalStatusResponse.class, response.getClass());
+                return;
+            }
+            try {
+                responseObserver.onNext((GlobalStatusResponseProto) ProtoTypeConvertHelper.convertToProto(response));
+                responseObserver.onCompleted();
+            } catch (Exception e) {
+                LOGGER.warn("[GRPC]fail to send response, req:{}, resp:{}", request, response);
+            }
+        });
+        remotingServer.processMessage(handleContext, requestModel);
+    }
+
+    public RpcMessageHandleContext buildHandleContext(StreamObserver<? extends Message> responseObserver, GrpcRemoting.BiStreamMessage message) {
+        RpcMessageHandleContext ctx = buildHandleContext(responseObserver);
+        if (null != message) {
+            MessageMeta messageMeta = new MessageMeta();
+            messageMeta.setMessageId(message.getID());
+            ctx.setMessageMeta(messageMeta);
+        }
+        return ctx;
+    }
+
+    public RpcMessageHandleContext buildHandleContext(StreamObserver<? extends Message> responseObserver) {
+        SeataChannel curChannel = new GrpcSeataChannel(CUR_CONNECT_ID.get(), CUR_CONNECTION.get(), responseObserver);
+        return new RpcMessageHandleContext(curChannel);
+    }
+
+    private boolean isProcessable(GrpcRemoting.BiStreamMessageType messageType) {
+        return true;
```

Comment:
add comment and add type judge

Reply:
Done. I think it is safer not to process messages by default here. And there is no need to do type checking, only the types that can be processed are released